src/shipping/consignment-action-creator.spec.ts

```diff
@@ -887,6 +888,19 @@ describe('consignmentActionCreator', () => {
         });
 
         it('sends request to update shipping address in first consignment', async () => {
+            jest.spyOn(store.getState().cart, 'getCart').mockReturnValue({
```

Comment:
It might be better to keep these 2 cases separate. One for just checking the payload get sent as is, one for _adding custom products to the payload_? What do you think?

Reply:
`updateShippingAddress` **always** sends the payload (address), **plus** all the shippable items in the cart. This used to be only physical, now it has been extended to include custom items.